.github/workflows/publish-docs-preview.yml

```diff
@@ -0,0 +1,58 @@
+name: Publish docs preview
+on:
+  pull_request_target:
+    types: labeled
+
+jobs:
+
+  publish-docs-preview:
+    name: Publish docs preview
+    runs-on: ubuntu-22.04
+    if: "${{ github.event.label.name == ':eyes: publish-docs-preview' }}"

```

Comment:
Had to quote the expression here, otherwise the `:` in the string literal is a syntax error... I hate YAML 😠